dali/operators/segmentation/random_object_bbox.h

```diff
@@ -192,7 +192,7 @@ class RandomObjectBBox : public Operator<CPUBackend> {
 
     ThreadPool *thread_pool = nullptr;
     TensorView<StorageCPU, int> out1, out2;
-    const Tensor<CPUBackend> *input = nullptr;
+    ConstSampleView<CPUBackend> input = {};
```

Comment:
```suggestion
    ConstSampleView<CPUBackend> input{};
```
I know it _should_ be the same, but, for some reason, I noticed that the compiler has a nasty habit of making a copy (in debug build).

Reply:
Removed.